deps/rabbit/src/rabbit_ssl.erl

```diff
@@ -109,28 +109,51 @@ peer_cert_subject_alternative_names(Cert, Type) ->
 peer_cert_validity(Cert) ->
     rabbit_cert_info:validity(Cert).
 
+-type ssl_cert_login_type() :: 
+    {subject_alternative_name | subject_alt_name, atom(), integer()} | 
+    {distinguished_name | common_name, undefined, undefined }.
+
+-spec extract_ssl_cert_login_settings() -> none | ssl_cert_login_type().
+extract_ssl_cert_login_settings() ->
+    case application:get_env(rabbit, ssl_cert_login_from) of 
+        {ok, Mode} ->
+            case Mode of 
+                subject_alternative_name -> extract_san_login_type(Mode);
+                subject_alt_name -> extract_san_login_type(Mode);
```

Comment:
Why do we allow both forms `subject_alternative_name ` and `subject_alt_name`?

Reply:
I wondered the same but  `rabbit_ssl:peer_cert_auth_name` supports both .. the same goes for `common_name`

Reply:
I have removed `subject_alt_name` from the schema. We are still sharing the internal type which still has it.